src/app/dashboard/weather-dashboard/weather-dashboard.component.html

```diff
@@ -1,2 +1,3 @@
+<qpac-filter-destination></qpac-filter-destination>
```

Comment:
We need to pass an output to the destination component in order to be able to pass the chosen city upwards..